frontend/packages/kubevirt-plugin/integration-tests/tests/vm.wizard.scenario.ts

```diff
@@ -133,22 +134,23 @@ describe('Kubevirt create VM using wizard', () => {
   it(
     'ID(CNV-3052) Creates DV with correct accessMode/volumeMode',
     async () => {
+      const expectedAccessMode = diskAccessMode.ReadWriteMany.value;
```

Comment:
I think we shouldn't assume that used storage class will support RWM. For example, hostpath-provisioner doesn't.
The reason for `resolveStorageDataAttribute` is to use accessMode and volume mode based on what is defined in `kubevirt-storage-class-defaults` CM.

I think we should add new test that selects accessMode and volumeMode to something other than is defined in the configMap and test that this setting is actually applied and that user is able to override the defaults (which is the point of this, because regular user don't have permissions to edit that configmap, and when they get a DV created by the UI, accessMode and volumeMode is not editable anymore)